input/definitions/datatypes/StructureDefinition-codeOptions.json

```diff
@@ -0,0 +1,102 @@
+{
+    "resourceType" : "StructureDefinition",
+    "id" : "codeOptions",
+    "extension" : [{
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+      "valueInteger" : 1
+    },
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+      "valueCode" : "cds"
+    },
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+      "valueCode" : "trial-use"
+    },
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-type-characteristics",
+      "valueCode" : "can-bind"
+    }],
+    "url" : "http://hl7.org/fhir/StructureDefinition/codeOptions",
+    "name" : "CodeOptions",
+    "title" : "Code Options",
+    "status" : "active",
```

Comment:
Possibly status = draft for publication of Ext Pack to enable publication of QI-Core in Feb 2025 using this extension.

Reply:
Pursuing with an FMM of 1, rather than status of draft